docs/release-notes/cloud/cloud-2024-08.md

```diff
@@ -0,0 +1,39 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud August 2024.
+included_jira_versions: ['2024.08']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.22.33
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.22.74
+---
+
+# Cloud August 2024
+
+These release notes are for the Codacy Cloud updates during August 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+-  Semgrep now supports PL/SQL rules from Checkmarx (TCE-942)
+-  Radon now calculates the average complexity of each file instead of calculating the highest complexity value of the file, which created some confusion (TCE-1091)
+-  Show status checks protection rule value everywhere in the UI (ALA-1160)
+-  We made several improvements on the code patterns page (ALA-1080)
+-  Now we support multiple coding standards (ALA-1119)
+-  Allow configuring repo-level Code Patterns while following a Coding Standard (ALA-1120)
+-  If you access the Issues page, you can see the issues grouped by patterns (ALA-1147)
+-  Now we support poetry.lock files when doing dependency checks with Trivy (TCE-1005)
```

Comment:
- Support of poetry.lock files when doing dependency checks with Trivy (TCE-1005)